apps/dashboard/app/views/batch_connect/session_contexts/_prefill_templates.html.erb

```diff
@@ -1,10 +1,10 @@
-<%- if !@prefill_templates.empty?() -%>
+<%- unless @prefill_templates.empty? -%>
   <div class="form-group">
     <label class="control-label"><%= t('dashboard.batch_connect_form_prefill') %></label>
     <select class="form-control selectpicker" id="batch_connect_session_prefill_template">
       <option disabled selected value> -- <%= t('dashboard.batch_connect_form_select_template') %> -- </option>
       <% @prefill_templates.each do |id, template| %>
-        <option value="<%= template %>"><%= id %></option>
+        <option value="<%= template %>"><%= id.humanize %></option>
```

Comment:
I'm unsure if humanizing the id's a good idea, as it's capitalizing the name that the user may not intend to have at all.

Reply:
We can remove it.

Reply:
Maybe we can keep this, as long as it's not about choosing the name.